ui/src/plugins/vue-i18n.js

```diff
@@ -30,4 +30,23 @@ Vue.use(VeeValidate, {
   }
 });
 
+export function getResponseErrorKey(statusCode) {
```

Comment:
Be sure to run Prettier before committing. This function looks great, but wasn't formatted properly.

Reply:
I've set up Prettier to auto-run now. This shouldn't be happening next time...